client/components/parts/ImageSearchField.jsx

```diff
@@ -28,63 +27,64 @@ export default class ImageSearchField extends Component {
   }
 
   handleChange = (e) => {
-    this.props.updateImage(e.target.id)
+    this.props.updateImage(e.target.dataset['url'])
```

Comment:
why not `dataset.url`? 
shouldn't `[<varname>]` be used only in the scenario where `varname` is dynamic?